src/components/TrackersManager.vue

```diff
@@ -0,0 +1,309 @@
+<script setup lang="ts">
+import { type ZodAffectType } from '@/types/zodAffect';
+import { useTrackers } from '@/composables/useTrackers';
+import { computed, toRef } from 'vue';
+
+const props = defineProps<{
+  flawId: string;
+  affects: ZodAffectType[];
+  mode: 'modal' | 'embedded';
+}>();
+
+const affects = toRef(props, 'affects');
+
+const {
+  trackerSelections,
+  trackersToFile,
+  fileTrackers,
+  toggleTrackerSelections,
+  unselectedStreams,
+  selectedStreams,
+  filterString,
+  isFilingTrackers,
+  alreadyFiledTrackers,
+  untrackableAffects,
+  isLoadingTrackers,
+} = useTrackers(props.flawId, affects);
+
+const availableAffectStreams = computed(() => {
+  const referenceAffect = props.affects;
+  return unselectedStreams?.value.filter((tracker) => {
+    return referenceAffect.some((refAffect) => {
+      return refAffect.ps_component === tracker.ps_component
+      && refAffect.ps_module === tracker.ps_module;
+    }
+    );
+  });
+});
+
+const selectedAffectStreams = computed(() => {
+  const referenceAffect = props.affects;
+  return selectedStreams?.value.filter((tracker) => {
+    return referenceAffect.some((refAffect) => {
+      return refAffect.ps_component === tracker.ps_component
+      && refAffect.ps_module === tracker.ps_module;
+    }
+    );
+  });
+});
+
+const emit = defineEmits<{
+  'affects-trackers:refresh': [];
+}>();
+
+function handleFileTrackers() {
+  fileTrackers().then(() => {
+    emit('affects-trackers:refresh');
+  });
+}
+</script>
+
+<template>
+  <div class="trackers-manager py-3 px-4" :class="{'embedded': mode === 'embedded'}">
+    <h4 v-if="mode === 'embedded'" class="mb-2 d-flex">
+      Trackers Manager
+    </h4>
+    <div class="d-flex gap-2">
+      <input
+        v-model="filterString"
+        type="text"
+        class="form-control form-control-sm"
+        style="width: 35%;"
```

Comment:
Do we want to use `style` attributes? or should we keep all styling inside the `style` tag?

Reply:
maybe use `ch` units for this field, like`25ch`